### PR TITLE
bcnews6 ticket 6

### PR DESCRIPTION
**Pull request by piknik1979**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -66,16 +65,38 @@ describe("GET /api/articles/:article_id", () => {
...
+      };
+      return request(app)
+        .patch("/api/articles/6")
+        .send(input)
+        .expect(400)
+        .then((res) => {
+          expect(res.body).toMatchObject({ message: "Bad request" });
+        });
+    });
+  });
```
Comment (jadelandeg) [inline __tests__/app.test.js:158]: Need more error testing here, what is the article id is invalid or not found?

**Review by jadelandeg — CHANGES_REQUESTED** [review]
need more error handling

File: __tests__/app.test.js (hunk 1 above)
Comment (piknik1979) [reply __tests__/app.test.js:158]: article_id actually is invalid here. hmm I shall still do another test. Thank you

Comment (jadelandeg) [reply __tests__/app.test.js:158]: if the article is not found, but the ID still a number then you should have 404 not found,
then you should have another test for an invalid article id (wrong data type like a string) which should be 400 invalid id

Comment (jadelandeg) [reply __tests__/app.test.js:121]: also think about what could happen if the update that you send is incorrect or has missing information

Comment (jadelandeg) [reply __tests__/app.test.js:158]: here just testing for invalid inc votes input which is good, also test for the other errors as written above

Comment (piknik1979) [reply __tests__/app.test.js:158]: Ouch, OK I got it I will do. Thanks

**Review by jadelandeg — CHANGES_REQUESTED** [review]
just a bit more error handling. I can just see you testing for invalid inc votes input, need to test for article-id mistakes too

**Review by giteron — APPROVED** [review]

**Pull request merged**